### DISCUSSION TITLE
Function suggestion: A configurable timer based announcer

### DISCUSSION THREAD

**Discussion by wolf109909** [ScureX/Takyon.PlayerVote#13] [Ideas]
Like an Advertisement display, it can show configurable messages with inserts like %time% and %playername% and such. so we can have something spamming in game like current time, server name, welcome messages 
example:
on left side of the screen it can show something like:
Welcome , %playername% !
You are now playing on "some server name"
Current time: %time%
join our community at “some link or name maybe”
for every few seconds or minutes maybe.
and like use this with a randomizer you can define like multiple messages to display, and each time the timer triggers to display something it can choose a random one from the defined messages. like displaying rules and some other advertisement shit maybe

**Comment by ScureX** [comment]
In v2.2.0 you can go to OnPlayerSpawned and change the spawn message. 
In v3.0.0 you can go to localization and change SPAWN_MESSAGE there.
You can edit those to get a welcome message when players first join (has already been done by some servers so i know it works). I like the idea of more customized messages tho and i will look into it!
